const throwError = (msg) => {
    throw new Error(msg);
};
export default throwError;
